### PR TITLE
feat: extend content

### PR DESCRIPTION
**Pull request by jdrueckert**

### CODE REVIEW

**Comment by jdrueckert** [comment]
As a preparation for our Code for Cause session, I took the liberty to go through the existing content, restructure some of it and add a bunch of new content to the AdventureSite.
I'm grateful for any kind of feedback :slightly_smiling_face:

File: src/data/Terasology/Development/index.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      name: "Yes, just show me open engine issues.",
+      link: 'https://github.com/MovingBlocks/Terasology/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Good+First+Issue%22',
+    },
+    {
+      name: "Yes, just show me open module issues.",
+      link: 'https://github.com/search?q=org%3ATerasology+is%3Apr+is%3Aissue&state=open&type=Issues'
+    },
+    {
+      name: 'Engine? Modules? Wait what?',
+      link: Architecture,
```
Comment (TheShubham99) [inline src/data/Terasology/Development/index.jsx:25]: This is an Intermediate Response Object I guess. 🤔 
```suggestion
      child: Architecture,
```

**Review by TheShubham99 — CHANGES_REQUESTED** [review]
Everything looks great 👍 
Thanks for the edit.

Just one change needed  
⬆️ I left a comment there. 😄

File: src/data/Terasology/Development/index.jsx (hunk 1 above)
Comment (jdrueckert) [reply src/data/Terasology/Development/index.jsx]: nice catch, thanks @TheShubham99 :+1:

**Review by Cervator — APPROVED** [review]
Great improvements, tests out well via GitPod 👍 

Lots more we can do of course but I'd be happy to see this merged as is.

Some potential next steps:

* Clear out or redirect a few things if obsolete or we can find more appropriate targets - the Collada page is obsolete (or not - we might need to discuss), and in some cases I wonder if it'd be better to link to (new?) tutorials (like for tech writing) than straight to the root of wikis and such. Yet everything is fine as is, this is more long term
* A few paths are odd, one about core game dev led back to the top of the site (rings a bell, pretty sure it precedes this PR)
* Most resources are naturally still Terasology focused, even including the options you get from the generic path (which should perhaps focus more on supporting / generic projects and roles? Dunno. Long term goals)
* Should we even _encourage_ IRC anymore? I usually idle there when awake, but am pretty hit or miss about catching the very rare question or two

**Review by TheShubham99 — APPROVED** [review]

**Pull request merged**